Event-Management-Application/event_management/src/main/java/com/example/event_management/model/LoginRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.example.event_management.model;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+public class LoginRequest {
+
+    @NotNull(message = "Username can not be empty")
```

Comment:
NotBlank checks for null and blank string as well
```suggestion
    @NotBlank(message = "Username can not be null or blank")
```